textile/api-docstrings.md

```diff
@@ -616,9 +616,106 @@ Enumerates the possible types of `Message` there can be.
+|| `type` || RTAN4b | A specific type string or an array of them to register the listener for. |
+|| `listener` || RTAN4a | An event listener function. |
+| unsubscribe() ||| RTAN5 | Deregisters all listeners currently receiving Annotation for the channel. |
+| unsubscribe(listener: (Annotation) ->)) ||| RTAN5 | Deregisters a specific listener that is registered to receive Annotation on the channel. |
+|| `listener` || RTAN5a | An event listener function. |
+| unsubscribe(String \| [String]) ||| RTAN5 | Deregisters any listener that is registered to receive Annotation on the channel for a specific type. |
+|| `type` || RTAN5a | A specific annotation type (or array of types) to deregister the listeners for. |
+| unsubscribe(String \| [String], listener: (Annotation) ->)) ||| RTAN5 | Deregisters a specific listener that is registered to receive Annotation on the channel for a given type. |
+|| `type` || RTAN5a | A specific annotation type (or array of types) to deregister the listener for. |
+|| `listener` || RTAN5a | An event listener function. |
+
+## Class Annotation
+
+| Property | Type | Spec | Description |
+|---|---|---|---|
+| id | string | TAN2a | An ID assigned (by the publisher or Ably) to this annotation, which is used as the idempotency key. |
+| action | AnnotationAction | TAN2b | The action, whether this is an annotation being added or removed, one of the AnnotationAction enum values. |
+| clientId | string? | TAN2c | The client ID of the publisher of this annotation (if any). |
+| name | string? | TAN2d | The name of this annotation. This is the field that most annotation aggregations will operate on. For example, using "distinct.v1" aggregation (specified in the type), the message summary will show a list of clients who have published an annotation with each distinct annotation.name. |
+| count | number? | TAN2e | An optional count, only relevant to certain aggregation types, see aggregation types documentation for more info. |
+| data | any? | TAN2f | An arbitrary publisher-provided payload, if provided. Not aggregated. |
+| encoding | string? | TAN2g | The encoding of the payload; typically empty as the data is decoded client-side back into the original data type. |
+| timestamp | number | TAN2h | Timestamp of when the annotation was received by Ably, as milliseconds since the Unix epoch. |
+| serial | string | TAN2i | This message's unique serial (lexicographically totally ordered). |
+| messageSerial | string | TAN2j | The serial of the message (of type `MESSAGE_CREATE`) that this annotation is annotating. |
+| type | string | TAN2k | The type of annotation it is, typically some name together with an aggregation method; for example: "emoji:distinct.v1". Handled opaquely by the SDK and validated serverside. |
```

Comment:
When I read this (not yet knowing anything about annotations), my first thought was that the "name" in "typically some name" was related to the `name` property of the annotation. Is there some other term we can use here to avoid this confusion?